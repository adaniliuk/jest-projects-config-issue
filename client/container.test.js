import React from 'react';
import { shallow } from 'enzyme';
import Container from './container';

describe('client/container.js', () => {
  it('renders container with the provided header and content', () => {
    const wrapper = shallow(
      <Container header="Test Example">
        Test Content goes here...
      </Container>
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('renders nothing if header is not defined', () => {
    const wrapper = shallow(
      <Container>
        Test Content goes here...
      </Container>
    );

    expect(wrapper.html()).toBeNull();
  });

  it('renders nothing if content is not defined', () => {
    const wrapper = shallow(
      <Container header="Test Example" />
    );

    expect(wrapper.html()).toBeNull();
  });
});
