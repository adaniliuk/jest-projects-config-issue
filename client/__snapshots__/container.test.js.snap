// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`client/container.js renders container with the provided header and content 1`] = `
<main>
  <h1>
    Test Example
  </h1>
  <section>
    Test Content goes here...
  </section>
</main>
`;
